### ISSUE TITLE
[Feature] Document jsdoc usage for contributors

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse-frontend#292]
## Problem

Generally, code contributors don't love types. We currently do not do anything to help make documenting types easier

## Description

Add a section to the README about typing Vue components with JSDoc. This article: https://medium.com/@znck/type-vue-without-typescript-b2b49210f0b seems to be a good description of some best practices.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by sarayourfriend** [comment]
@zackkrida I'm not sure exactly what would be the best format for these docs... and I wouldn't want to duplicate any existing information.

Perhaps the TypeScript documentation on JSDoc would be a good thing to link as a resource for people to take a look at? https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

**Comment by dhruvkb** [comment]
Yeah, we should point to the relevant docs and official guides rather than writing our own.

**Comment by zackkrida** [comment]
I should have been clearer in my issue description, I'm not looking to explain the spec of JSDoc, rather to include information that this codebase _uses_ JSDoc and that JSDoc types should be written by PR authors. This could be a section in the README or elsewhere, I suppose.

I also thought that Vue-specific documentation could be useful for contributors, whether we link to that external file or write our own summary.
